internal/cloud/api.go

```diff
@@ -160,6 +160,42 @@ func (a *api) GetClusterDetail(clusterID cloud.ID) (*cloud.Cluster, error) {
 	return cluster, nil
 }
 
+func (a *api) ListServices(clusterID cloud.ID, limit int, skip int) ([]*cloud.Application, error) {
+	var services []*cloud.Application
+	pageSize := limit
+	page := int(math.Floor(float64(skip)/float64(limit))) + 1
+	start := skip % limit
+	end := skip%limit + limit
```

Comment:
It looks complicated.

Why not use a fixed page size, e.g., 25

Then:

```
startPage := skip / pageSize
startPageSkip = pageSize - skip % pageSize
```

Reply:
fixed

Reply:
fixed